mr/src/main/java/org/opensearch/hadoop/rest/commonshttp/CommonsHttpTransport.java

```diff
@@ -214,7 +221,8 @@ public CommonsHttpTransport(Settings settings, SecureSettings secureSettings, St
         }
         this.settings = settings;
         this.secureSettings = secureSettings;
-        this.clusterName = settings.getClusterInfoOrUnnamedLatest().getClusterName().getName(); // May be a bootstrap client.
+        this.clusterName = settings.getClusterInfoOrUnnamedLatest().getClusterName().getName(); // May be a bootstrap
```

Comment:
is it possible to revert code format related change? a lot of code format change make it hard to tell what is the required change.
we can enforce the format in another PR.